grantnav/frontend/tests_functional.py

```diff
@@ -533,9 +533,9 @@ def test_links(provenance_dataload, server_url, browser, path):
         except KeyError:
             try:
                 # Some sites reject connection without a user agent
-                r = requests.head(link, headers={"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"})
+                r = requests.get(link, headers={"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"})
```

Comment:
I think we'd stick with HEAD as this should be much faster than actually retrieving the page content